cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,32 @@
-describe('', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
 
-  });
+const ArticlePage = new ArticlePageObject();
 
-  it('', () => {
+describe('Article', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user.email, user.username, user.password);
+      cy.visit('/editor');
+    });
   });
 
-  it('', () => {
+  it('should fill in article and create it', () => {
+    const article = ArticlePage.generateArticle();
+    ArticlePage.createArticle(article);
+    ArticlePage.onClickSubmitArticle();
```

Comment:
The method `onClickSubmitArticle` should be defined in the `ArticlePageObject`. Ensure it triggers the article submission correctly.